<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reserve</class>
 <widget class="QMainWindow" name="Reserve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>756</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>756</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reserve a Room</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f3e8d5;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Name_R">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Surename_R">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Surename</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Email_R">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Passport">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Passport or ID details:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Checkout">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: #bc663f;
 color: White
}
QPushButton:hover {
background-color: #e6a081; 
color: White;              
}</string>
    </property>
    <property name="text">
     <string>Checkout</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Kitchen_Option">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>---Kitchen Options---</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Breakfast Included - $10/day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Breakfast and Lunch - $20/day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All-Inclusive - $50/day</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Pet">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>Pet?</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(/home/user/Desktop/hotel_project/Hotel/images/Standard.jpg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(/home/user/Desktop/hotel_project/Hotel/images/Premium.jpg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(/home/user/Desktop/hotel_project/Hotel/images/Luxury1.jpg);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="StandardRoom">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Standard Room - $120/night</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="PremiumRoom">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Premium Room - $180/night</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LuxuryRoom">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Luxury Room - $250/night</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(119, 118, 123);</string>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <zorder>Surename_R</zorder>
   <zorder>Email_R</zorder>
   <zorder>Passport</zorder>
   <zorder>Checkout</zorder>
   <zorder>Kitchen_Option</zorder>
   <zorder>Pet</zorder>
   <zorder>Name_R</zorder>
   <zorder>widget</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget_2</zorder>
   <zorder>StandardRoom</zorder>
   <zorder>PremiumRoom</zorder>
   <zorder>LuxuryRoom</zorder>
   <zorder>BackButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
